governance/proposals/up/002-change-voting-duration.js

```diff
@@ -0,0 +1,29 @@
+const { UPGovernor, UPTimelock } = require('@unlock-protocol/contracts')
+
+const BASE_UP_GOVERNOR_ADDRESS = '0x65bA0624403Fc5Ca2b20479e9F626eD4D78E0aD9'
+const BASE_UP_TIMELOCK_ADDRESS = '0xB34567C4cA697b39F72e1a8478f285329A98ed1b'
```

Comment:
It is better to get the time lock address through the gov contract instead of hardcoding it to prevent possible mistakes